### PR TITLE
EthRelay module tests

### PR DESCRIPTION
**Pull request by HackFisher**

### CODE REVIEW

File: srml/eth-relay/src/lib.rs [hunk 1]
```diff
@@ -25,6 +25,9 @@ use merkle_patricia_trie::{trie::Trie, MerklePatriciaTrie, Proof};
 
 type DAG = LightDAG<EthereumPatch>;
 
+mod mock;
```
Comment (AurevoirXavier) [inline srml/eth-relay/src/lib.rs]:
```rust
#[cfg(test)]
```

File: srml/eth-relay/src/lib.rs [hunk 2]
```diff
@@ -25,6 +25,9 @@ use merkle_patricia_trie::{trie::Trie, MerklePatriciaTrie, Proof};
...
+mod tests;
```
Comment (AurevoirXavier) [inline srml/eth-relay/src/lib.rs]:
```rust
#[cfg(test)]
```

File: srml/eth-relay/src/tests.rs [hunk 3]
```diff
@@ -0,0 +1,111 @@
+//! Tests for the module.
+
+#![cfg(test)]
```
Comment (AurevoirXavier) [inline srml/eth-relay/src/tests.rs]: why put it here

File: srml/eth-relay/src/mock.rs [hunk 4]
```diff
@@ -0,0 +1,155 @@
+//! Test utilities
+
+#![cfg(test)]
```
Comment (AurevoirXavier) [inline srml/eth-relay/src/mock.rs]: why put it here

Comment (HackFisher) [reply srml/eth-relay/src/mock.rs]: I copied from the balances module tests

**Pull request merged**